### ISSUE TITLE
Allow screen reader to read day when using arrow navigation

### ISSUE THREAD

**Issue by ExplosiveHippo** [angular-ui/bootstrap#6515]
### Bug description:
Screen reader does not read day when navigating through calendar with arrow keys

### Steps to reproduce the issue: 
Turn on voice over, tab into the date picker widget, use the arrow keys to navigate to a new day in the month. Voiceover will not read the day as you're navigating through it.

### Version of Angular, UIBS, and Bootstrap

Angular: v1.5.11

UIBS: v2.5.0

Bootstrap: v3.3.7
